pbm/log/log.go

```diff
@@ -332,3 +332,64 @@ func Get(cn *mongo.Collection, r *LogRequest, limit int64, exactSeverity bool) (
 
 	return e, nil
 }
+
+func Follow(ctx context.Context, coll *mongo.Collection, r *LogRequest, exactSeverity bool) (<-chan *Entry, <-chan error) {
+	filter := bson.D{bson.E{"s", bson.M{"$lte": r.Severity}}}
+	if exactSeverity {
+		filter = bson.D{bson.E{"s", r.Severity}}
+	}
+
+	if r.RS != "" {
+		filter = append(filter, bson.E{"rs", r.RS})
+	}
+	if r.Node != "" {
+		filter = append(filter, bson.E{"node", r.Node})
+	}
+	if r.Event != "" {
+		filter = append(filter, bson.E{"e", r.Event})
+	}
+	if r.ObjName != "" {
+		filter = append(filter, bson.E{"eobj", r.ObjName})
+	}
+	if r.Epoch.T > 0 {
+		filter = append(filter, bson.E{"ep", r.Epoch})
+	}
+	if r.OPID != "" {
+		filter = append(filter, bson.E{"opid", r.OPID})
+	}
+	if !r.TimeMin.IsZero() {
+		filter = append(filter, bson.E{"ts", bson.M{"$gte": r.TimeMin.Unix()}})
+	}
+	if !r.TimeMax.IsZero() {
+		filter = append(filter, bson.E{"ts", bson.M{"$lte": r.TimeMax.Unix()}})
+	}
```

Comment:
I'd move this part for some `func filter(r *LogRequest, exactSeverity bool) bson.D`. As the same code is used in `Get()` and may lead to some errors should we change it (add some filter for example).